### PR TITLE
Improve prediction algorithm correctness by 100%

### PR DESCRIPTION
**Pull request by alexblackie**

### CODE REVIEW

**Comment by rtandy** [comment]
:+1:

**Comment by Sinetheta** [comment]
DENIED WITH PREJUDICE it's on

**Comment by jhawthorn** [comment]
:+1:

**Comment by alexblackie** [comment]
THIS IS AGAINST THE SPIRIT OF OPEN SOURCE!

This version improves performance, decreases (i.e., eliminates) build time, and is much easier to deploy and host. This PR is obviously an improvement over the existing implementation and your hasty dismissal is truly a disappointment to myself and the entire open source community.

**Comment by Sinetheta** [comment]
denied

**Comment by alexblackie** [comment]
repressed

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<head>
+	<meta charset="UTF-8">
+	<title>Should I Order a Pizza?</title>
+</head>
+<body>
+	<center>
```
Comment (Sinetheta) [inline index.html]: [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) gtfo

**Pull request closed without merging**